src/ontology/cl-edit.owl

```diff
@@ -32633,6 +32636,14 @@ AnnotationAssertion(terms:date obo:CL_4047019 "2024-09-24T12:11:09Z"^^xsd:dateTi
 AnnotationAssertion(rdfs:label obo:CL_4047019 "early enterocyte"@en)
 EquivalentClasses(obo:CL_4047019 ObjectIntersectionOf(obo:CL_0000584 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0013740)))
 
+# Class: obo:CL_4047025 (type G cell of stomach)
```

Comment:
could you please add a brief definition and references to support the logical defs - this could be a separate ticket.